setup.py

```diff
@@ -34,6 +34,11 @@
     install_requires=[
         'django>=1.8,!=2.1.0,!=2.1.1,<2.2',
     ],
+    extras_require={
+        'lock': [
+            'python-redis-lock>=3.2.0',
```

Comment:
Might want to add `redis` here as well. For future we might want to create extra for hash models as well and should be done later.

Reply:
I probably over thought this, as `python-redis-lock` depends on redis already, we might have not needed it here.